src/types/markers.js

```diff
@@ -500,6 +500,7 @@ export type UserTimingMarkerPayload = {|
 export type TextMarkerPayload = {|
   type: 'Text',
   name: string,
+  cause?: CauseBacktrace,
```

Comment:
Hm, I think it would be better to make these changes in another/earlier commit. Currently it's inside the commit that we add the test.